### ISSUE TITLE
gislab-moduser administrator command

### ISSUE THREAD

**Issue by imincik** [imincik/gis-lab#461]
gislab-moduser administrator command which will be capable to modify all information stored with user account. When implemented, remove gislab-password command.

**Comment by imincik** [comment]
@mpastor  and @landam (in cc),
gislab-moduser is breaking account customization hooks. See following output from tests:
```
<127.0.0.1> REMOTE_MODULE command gislab-deluser -f "testsu49ac4d6f1df143eca5679b07" #USE_SHELL
failed: [gislab_vagrant] => (item=testsu49ac4d6f1df143eca5679b07) => {"changed": false, "cmd": "gislab-deluser -f \"testsu49ac4d6f1df143eca5679b07\"", "delta": "0:00:00.140363", "end": "2015-11-11 22:49:45.722707", "item": "testsu49ac4d6f1df143eca5679b07", "rc": 255, "start": "2015-11-11 22:49:45.582344", "warnings": []}
stdout: [GIS.lab]: Running 'before-delete' account hooks.
[GIS.lab]: Deleting user account.
Successfully deleted user testsu49ac4d6f1df143eca5679b07 from group gislabadmins
[GIS.lab]: User 'testsu49ac4d6f1df143eca5679b07' was successfully deleted from superuser's group.
USAGE: gislab-moduser [OPTIONS] username
Modify GIS.lab user account.

OPTIONS
        -a add user to supplementary group(s), separate groups with comma
        -A delete user from supplementary group(s), separate groups with comma
        -s add user to superuser's group
        -S delete user from superuser's group
        -m change user email
        -p change user password (if password is not given, it will be generated)
        -d change user description
        -h display this help

...ignoring
```

**Comment by imincik** [comment]
@mpastor , please add also some tests.

**Comment by imincik** [comment]
@mpastor, please replace queries to users database (using id command and similar) with ldapsearch

**Comment by imincik** [comment]
@mpastor, please update tests once again.

**Comment by mpastor** [comment]
Closed in 62fd7fe.
